app/src/main/java/com/keylesspalace/tusky/AccountActivity.java

```diff
@@ -91,6 +94,8 @@
     public AccountManager accountManager;
     @Inject
     public DispatchingAndroidInjector<Fragment> dispatchingAndroidInjector;
+    @Inject
+    public AppStore appstore;
```

Comment:
Is there a better name for this? It just reminds me so much of fdroid/google play 🙈

Reply:
It's a Redux/Flux terminology to name such a thing Store. Can rename to GlobalStore or something